Library/DataSource/ValueCellDataSource.swift

```diff
@@ -71,21 +71,45 @@ open class ValueCellDataSource: NSObject, UICollectionViewDataSource, UITableVie
   }
 
   /**
+   Inserts a single value at the beginning of the section specified.
+
+   - parameter value:     A value to prepend.
+   - parameter cellClass: The type of cell associated with the value.
+   - parameter section:   The section to append the value to.
+
+   - returns: The index path of the prepended row.
+   */
+  @discardableResult
+  public final func prependRow <
```

Comment:
Added `prependRow` for inserting the chat messages at the beginning each time. Also for animating the rows in we require the `IndexPath`. Despite this always being `row: 0, section: 0` figured it's best to return it here and on `appendRow` for consistency.